spatial-backdrop/README.md

```diff
@@ -0,0 +1,129 @@
+# Spatial Backdrop Explainer
+
+The purpose of this document is to explain a new [Link type](https://html.spec.whatwg.org/#linkTypes) for adding Immersive Backdrops from 3D models to Web Pages. The primary target for this is for standalone head mounted immersive displays but it should have application on other platforms too.
+
+## Authors
+
+* [Ada Rose Cannon](https://github.com/adarosecannon)
+* [Jean Haberer](https://github.com/jeanhbr)
+
+## Background
+
+Operating systems on virtual reality devices have the capability of displaying virtual environments behind the windows and volumes of the various applications running on the device including its Web Browsers.
+This new Link type enables Websites to to change this virtual environment to one that is specified by the Web Page.
+
+![An animation of a user in front of a stylized web browser, a backdrop environment is revealed around them.](./assets/backdropanim.webp)
+*Model Source: [Kraków's bed by on sketchfab by Wirtualne Muzea Małopolski](https://sketchfab.com/WirtualneMuzeaMalopolski/collections/national-museum-in-krakow-5bf7a88a295848fa886898a9335d4aa7)*
+
+## Use Cases
+
+* Website Ambiance, E.g. look at the page for a popular streaming series in an environment from that series. 
+* Displaying life-sized Products such as a Car Showroom, Hotel Room
+* Informative, see these dinosaurs to scale as you read about the jurassic era.
+
+Developers should not assume the user or UA will always display the backdrop they specified. Like theme color it has an effect outside of the web page but isn’t guaranteed to be present. This feature works best for bonus content as the page will not know if it is supported or currently active.
+
+## Concepts
+
+This capability is for displaying full scale 3D models as virtual environments. Since 3D model formats are quite flexible this could have a wide variety of appearance such as hand crafted polygon scenes, scans of real world environments or simple geometry with photographic textures applied.
+This is related to the HTML model element as it is also used for displaying 3D models in the context of a Web Page, although this places the 3D content around the user rather than embedding it into a web page.
```

Comment:
```suggestion
This is related to the HTML model element as it is also used for displaying 3D models in the context of a web page, although this places the 3D content around the user rather than embedding it into a web page.
```